src/story.ts

```diff
@@ -63,6 +63,18 @@ interface ControlBase {
   disable?: boolean;
 }
 
+interface Report {
+  id: string;
```

Comment:
Should we rename `id` to `type` instead? Does it make more sense? @ghengeveld raised his voice and said that an `id` should be a unique identifier per report instead of being used to distinguish between separate reporter types.
@kasperpeulen